### PR TITLE
Bug Slaughter

### PR DESCRIPTION
**Pull request by asanchez0527**
Fixed a bunch of bugs over the page

### CODE REVIEW

**Comment by RogelioMart** [comment]
I tested the sign up feature, making events, and the mailers.
I tried to sign up using two emails both emails threw me an error called Username and Password not accepted. (check the attached image to see the error I got.) Despite this both emails were saved into the development database and it said that the confirmation email was sent. I waited a few minutes and did checked other things on my local host but I never received a confirmation email (I did check my spam folder.) In order to sign in I faked the confirmation_token and confirmed_at by using the following sql code.



Once I was able to sign in I had no problem creating study hall, conversation, and group problem solving events. I also rsvp to the events I created but got no emails telling me I created these events or that I rsvp to them. 

I did check my development.rb file and it was the same as the one that is in the README.md file. To double check this mailer problem I ran my own code on the local host and made new events but I did not receive any emails either.

There are still some bugs that need to be debugged but I would like for another person to check this test branch to see if they can replicate my errors.

userNameError

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def show
     @rsvpusers = @event.users
     @rsvps     = @event.rsvpqs
     @duration  = ((@event.end_at - @event.start_at) / 60).floor
-    @surl = request.base_url + "/" + @user.permalink
+    @surl = "https://www.think.tv" + "/" + @user.permalink
```
Comment (Hieu-Minh-Phung) [inline app/controllers/events_controller.rb:19]: I think this should be https://thinq.tv

**Comment by Hieu-Minh-Phung** [comment]
I used rails test command for SQL logs issue and it seems they successfully fixed this.
When testing on localhost:3000, I tried to sign up using 2 different emails on both Firefox and Chrome and both received 2 emails each time (one welcome email and one confirmation instruction). It seems sign up is okay for me, no error.

I had no issues with creating study hall, conversation, and group problem solving events. 

I also create a future event and rsvp but received no email.

After this, I created a live conversation and joined. From there, I checked in if all links can be open in new tabs when I'm in the show page and they all opened in new tabs as expected.

Then I sign out of my account and tried to join in the live Conversation I just created. I was rerouted to https://thinq.tv/ but not the localhost, without any alert popping up.

All of the processes above were tested in both Firefox and Chrome.

**Comment by RogelioMart** [comment]
I changed my development.rb to use double quotes and I can now sign up without any errors. I can also create events for which I receive an email but when i rsvp to an event I do not receive an email.

**Pull request merged**